build-system/tasks/visual-diff/index.js

```diff
@@ -57,6 +57,10 @@ const percyCss = [
   '.i-amphtml-new-loader * { animation: none !important; }',
 ].join('\n');
 
+// Fix Chromium version to 78.0.3904.0, same as the one running on Percy.
```

Comment:
Can we add a TODO to change / remove this when Percy updates their version of Chrome?

Reply:
`REPEATING TODO`?